cursorless-talon/src/spoken_scope_forms.py

```diff
@@ -0,0 +1,76 @@
+from talon import Context
+
+from .csv_overrides import csv_get_ctx, csv_get_normalized_ctx
+
+
+def init_scope_spoken_forms(graphemes_talon_list: dict[str, str]):
+    create_flattened_talon_list(csv_get_ctx(), graphemes_talon_list, True)
+    create_flattened_talon_list(csv_get_normalized_ctx(), graphemes_talon_list, False)
+
+
+def create_flattened_talon_list(
+    ctx: Context, graphemes_talon_list: dict[str, str], include_custom_regex: bool
+):
+    scope_types_singular: dict[str, str] = {}
+    scope_types_plural: dict[str, str] = {}
+
+    append_talon_lists(
+        ctx, scope_types_singular, scope_types_plural, "simple", "cursorless_scope_type"
+    )
+    append_talon_lists(
+        ctx,
+        scope_types_singular,
+        scope_types_plural,
+        "surroundingPair",
+        "cursorless_surrounding_pair_scope_type",
+    )
+    append_talon_lists(
+        ctx,
+        scope_types_singular,
+        scope_types_plural,
+        "surroundingPair",
+        "cursorless_selectable_only_paired_delimiter",
+    )
+    append_talon_lists(
+        ctx,
+        scope_types_singular,
+        scope_types_plural,
+        "surroundingPair",
+        "cursorless_wrapper_selectable_paired_delimiter",
+    )
+    if include_custom_regex:
```

Comment:
The default grammar have no custom regexes and that would throw an error when we couldn't find the list in the register